Source/WebCore/css/parser/CSSPropertyParserConsumer+LengthPercentage.cpp

```diff
@@ -165,12 +165,13 @@ std::optional<CSS::LengthPercentageRaw> LengthPercentageKnownTokenTypeNumberCons
 
 // MARK: - Consumer functions
 
-RefPtr<CSSPrimitiveValue> consumeLengthPercentage(CSSParserTokenRange& range, const CSSParserContext& context, ValueRange valueRange, UnitlessQuirk unitless, UnitlessZeroQuirk unitlessZero, NegativePercentagePolicy negativePercentage, AnchorPolicy anchorPolicy)
+RefPtr<CSSPrimitiveValue> consumeLengthPercentage(CSSParserTokenRange& range, const CSSParserContext& context, ValueRange valueRange, UnitlessQuirk unitless, UnitlessZeroQuirk unitlessZero, NegativePercentagePolicy negativePercentage, AnchorPolicy anchorPolicy, AnchorSizePolicy anchorSizePolicy)
```

Comment:
One more policy sigh. I don't particularly like it, but unfortunately the properties that accept `anchor-size()` is a superset of the properties that accept `anchor()` (i.e some property accepts `anchor-size()` but not `anchor()`), hence the additional policy.

Reply:
Antti said it's okay to have a policy here. It's high time we made `consumeLengthPercentage` take `CSSPropertyParseOptions` though.